app/plugin.go

```diff
@@ -638,6 +625,83 @@ func (a *App) notifyPluginEnabled(manifest *model.Manifest) error {
 	return nil
 }
 
+func (a *App) installPrepackagedPlugins(pluginsDir string) []*plugin.PrepackagedPlugin {
+	prepackagedPluginsDir, found := fileutils.FindDir(pluginsDir)
+	if !found {
+		return nil
+	}
+
+	fileStorePaths := []string{}
+	if err := filepath.Walk(prepackagedPluginsDir, func(walkPath string, info os.FileInfo, err error) error {
+		fileStorePaths = append(fileStorePaths, walkPath)
+		return nil
+	}); err != nil {
+		mlog.Error("Failed to walk prepackaged plugins", mlog.Err(err))
+	}
+	pluginSignaturePathMap := getPluginsFromFilePaths(fileStorePaths)
+	plugins := make([]*plugin.PrepackagedPlugin, 0, len(pluginSignaturePathMap))
+	for _, pluginPaths := range pluginSignaturePathMap {
+		plugin, err := a.installPrepackagedPlugin(pluginPaths)
+		if err != nil {
+			mlog.Error("Failed to install prepackaged plugin %s", mlog.Err(err), mlog.String("path", pluginPaths.path))
+			continue
+		}
+		plugins = append(plugins, plugin)
+	}
+	return plugins
+}
+
+func (a *App) installPrepackagedPlugin(pluginPaths *pluginSignaturePaths) (*plugin.PrepackagedPlugin, error) {
+	fileReader, err := os.Open(pluginPaths.path)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Failed to open prepackaged plugin %s", pluginPaths.path)
+	}
+	tmpDir, err := ioutil.TempDir("", "plugintmp")
+	if err != nil {
+		return nil, errors.Wrapf(err, "Failed to create temp dir plugintmp")
+	}
+	defer os.RemoveAll(tmpDir)
+
+	plug, pluginDir, err := getPrepackagedPlugin(pluginPaths, fileReader, tmpDir)
```

Comment:
We use `plugin` all the time. I would like to stay consistent.
```suggestion
	plugin, pluginDir, err := getPrepackagedPlugin(pluginPaths, fileReader, tmpDir)
```